### PR TITLE
Skinnern/hard mode class

### PR DESCRIPTION
**Pull request by Skinnern**
This contains the much anticipated hard mode class, I know many of you have been waiting for this change for about 2 weeks, but that wait is almost over.

In this branch I also removed a legacy println I had for testing purposes, I believe that all should be in working order though.

### CODE REVIEW

File: src/main/java/models/HardMode.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package models;
+
+public class DefaultGame extends Game {
```
Comment (danvanhorn) [inline src/main/java/models/HardMode.java]: This class should be called HardGame because Bailey is creating the DefaultGame class.

Comment (danvanhorn) [reply src/main/java/models/HardMode.java]: You should also name the file the same name as the class, `HardGame.java` as opposed to `HardMode.java`. I believe that is the best practice for Java.

Comment (baileysingleton) [reply src/main/java/models/HardMode.java]: ^^

**Comment by Skinnern** [comment]
@danvanhorn good eye! I forgot to push that change after making the pull request. I have made the requested change, and updated the branch.

**Comment by baileysingleton** [comment]
dibs merging

**Comment by Skinnern** [comment]
@baileysingleton Done and done

**Comment by baileysingleton** [comment]
my man! good to go from me

**Comment by cashd932** [comment]
I pushed up a quick change -- the move conditional was comparing the card object to an int rather than the card's value. I'll go ahead and merge

**Pull request merged**